File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusGameTournamentBatchStatusForm.kt
package com.sdkwork.backend

data class PlusGameTournamentBatchStatusForm(
    val userIds: List<Int>? = null,
    val playerItems: List<PlusGameTournamentBatchStatusPlayerForm>? = null,
    val status: String? = null,
    val startRankNo: Int? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_game_tournament_batch_status_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusGameTournamentBatchStatusForm:
    """棋牌游戏赛事参赛者批量状态管理表单"""
    user_ids: List[int] = None
    player_items: List[PlusGameTournamentBatchStatusPlayerForm] = None
    status: str = None
    start_rank_no: int = None
